gpkit/keydict.py

```diff
@@ -170,3 +173,8 @@ def __getitem__(self, key):
     def __setitem__(self, key, value):
         "Assigns the key itself every time."
         KeyDict.__setitem__(self, key, None)
+
+
+class FastKeyDict(KeyDict):
+    "FsatKeyDicts are KeyDicts that don't map keys, only collate arrays"
```

Comment:
Re: when to use, yeah I should clarify.

Basically, KeyDicts have two features:
  1. access values with alternate keys
  2. collapse array-keyed values into the appropriate arrays

KeySets have only the first of those, FastKeyDicts have only the second.